types/office-js/index.d.ts

```diff
@@ -93500,813 +94214,657 @@ declare namespace Visio {
     }
     /**
      *
-     * Represents the Shape class.
+     * Master Information
      *
      * [Api set:  1.1]
      */
-    class Shape extends OfficeExtension.ClientObject {
-        /** The request context associated with the object. This connects the add-in's process to the Office host application's process. */
-        context: RequestContext; 
-        /**
-         *
-         * Returns the Comments Collection. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly comments: Visio.CommentCollection;
-        /**
-         *
-         * Returns the Hyperlinks collection for a Shape object. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly hyperlinks: Visio.HyperlinkCollection;
-        /**
-         *
-         * Returns the Shape's Data Section. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly shapeDataItems: Visio.ShapeDataItemCollection;
-        /**
-         *
-         * Gets SubShape Collection. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly subShapes: Visio.ShapeCollection;
-        /**
-         *
-         * Returns the view of the shape. Read-only.
-         *
-         * [Api set:  1.1]
-         */
-        readonly view: Visio.ShapeView;
+    interface MasterInfo {
         /**
          *
-         * Shape's identifier. Read-only.
+         * Localised Display Name of the Master
```

Comment:
Perhaps should be something like this?
```suggestion
         * Localized display name of the master.
```